src/cljs/replumb/load.cljs

```diff
@@ -68,32 +85,43 @@
   tries to load the unevaluated ones.
   This function does not check whether parameters are nil, please do it in
   the caller."
-  [verbose? file-names read-file-fn! load-fn-cb cached-file-names]
+  [verbose? file-names read-file-fn! load-fn-cb cached-file-names name]
   (if-let [[js-path cache-path] (first cached-file-names)]
     (let [try-next-files-pair #(read-files-from-cache-and-callback! verbose?
                                                                     file-names
                                                                     read-file-fn!
                                                                     load-fn-cb
-                                                                    (rest cached-file-names))]
-      (when verbose?
-        (common/debug-prn "Reading" js-path "..."))
-      (read-file-fn! js-path
-                     (fn [js-source]
-                       (if (and js-source (cache/cached-js-valid? js-source))
-                         (do
-                           (when verbose?
-                             (common/debug-prn "Reading" cache-path "..."))
-                           (read-file-fn! cache-path
-                                          (fn [cache-source]
-                                            (if cache-source
-                                              (do
-                                                (when verbose?
-                                                  (common/debug-prn (str "Retrieved from cache " js-path " and " cache-path)))
-                                                (load-fn-cb {:lang (filename->lang js-path)
-                                                             :source js-source
-                                                             :cache (read-cache-source cache-path cache-source)}))
-                                              (try-next-files-pair)))))
-                         (try-next-files-pair)))))
+                                                                    (rest cached-file-names)
+                                                                    name)
+          read-cache-file (fn [js-source]
```

Comment:
What about creating an external function, reducing the lines of code for this one